### DISCUSSION TITLE
EBSD data import as csv or txt

### DISCUSSION THREAD

**Discussion by Sarah-ol** [mtex-toolbox/mtex#1459] [Ask Anything] [answered]
Hello,

I've been trying to import my EBSD data (as csv or txt) using import_wizard but I keep having the "could not detect file format" error message. I know that's not its preferred format so I tried to follow this link https://mtex-toolbox.github.io/loadEBSD_generic.html but no luck as well. Could you please help me load these data? My goal is to obtain an orientation map of my olivine grains.

Here's my code:
fname = fullfile(mtexDataPath,'EBSD','test3.csv');
CS = crystalSymmetry('mmm', [4.756 10.207 5.98], 'mineral', 'Forsterite');
SS = specimenSymmetry('orthorhombic');
ebsd = loadEBSD_csv(fname,'CS',CS,'SS',SS, 'ColumnNames',{'phi1','PHI','phi2','x','y'}, 'Bunge');

Please find attached my csv file (test3).
[test3.csv](https://github.com/mtex-toolbox/mtex/files/8925288/test3.csv)

Thank you in advance,

Sarah

**Comment by kilir** [comment, marked as answer]
Hi,
try:
```
ebsd = loadEBSD_generic('test3.csv', 'CS', CS, 'ColumnNames', ...
      {'phi1' 'PHI' 'phi2' 'X' 'Y'}, 'Bunge')
```
Cheers,
Rüdiger

**Reply by Sarah-ol** [reply]
Thank you, it works!

**Reply by Sarah-ol** [reply]
Another question - When I get to this stage: 
plotx2east % this command tells MTEX to plot the x coordinates increasing to the east (left)

How can I tell MTEX to plot the y-axis in reverse? What I mean is that I'm seeing the image upside down (x is in the right direction)

Thank you!
Sarah

**Reply by kilir** [reply]
Have a look here https://mtex-toolbox.github.io/plotx2east.html also at the other functions 'plotx2...' and 'plotz...'. Mtex uses a right handed coordinate system.
Cheers,
Rüdiger
